config/jobs/kubernetes/sig-windows/windows-gce.yaml

```diff
@@ -200,7 +202,7 @@ periodics:
     preset-service-account: "true"
     preset-common-gce-windows: "true"
     preset-e2e-gce-windows: "true"
-    preset-windows-repo-list-2004: "true"
+    preset-windows-repo-list-master: "true"
```

Comment:
I think we need to keep this one at 2004: https://github.com/kubernetes/test-infra/blob/3e9f5cf5f61318e03de3ffc0e16af8926b658b49/config/jobs/kubernetes-sigs/sig-windows/sig-windows-sac.yaml#L120

is that right @chewong ?

Reply:
reviewed at the same time ignore this...